lib/android-tools-info.ts

```diff
@@ -299,22 +312,7 @@ export class AndroidToolsInfo implements NativeScriptDoctor.IAndroidToolsInfo {
 	}
 
 	private getSystemRequirementsLink(): string {
-		let linkToSystemRequirements: string;
-		switch (process.platform) {
-			case "linux":
-				linkToSystemRequirements = "http://docs.nativescript.org/setup/ns-cli-setup/ns-setup-linux.html#system-requirements";
-				break;
-			case "win32":
-				linkToSystemRequirements = "http://docs.nativescript.org/setup/ns-cli-setup/ns-setup-win.html#system-requirements";
-				break;
-			case "darwin":
-				linkToSystemRequirements = "http://docs.nativescript.org/setup/ns-cli-setup/ns-setup-os-x.html#system-requirements";
-				break;
-			default:
-				linkToSystemRequirements = "";
-		}
-
-		return linkToSystemRequirements;
+		return Constants.SYSTEM_REQUIREMENTS_LINKS[process.platform] || "";
```

Comment:
Are we sure about the casing of this property?

Reply:
Yes:
https://nodejs.org/dist/latest-v9.x/docs/api/process.html#process_process_platform